README.md

```diff
@@ -280,6 +280,12 @@ This would produce api doc something similar to:
 - More about OpenAPI security in [security documentation](https://docs.rs/utoipa/latest/utoipa/openapi/security/index.html).
 - Dump generated API doc to file at build time. See [issue 214 comment](https://github.com/juhaku/utoipa/issues/214#issuecomment-1179589373).
 
+## General Pitfalls
+
+### SwaggerUI not working inside Docker
+
+When running your Rust application with SwaggerUI inside a Docker container, it's only possible to use a release build. This is because the [debug-embed](https://github.com/pyrossh/rust-embed#features) flag is not enabled in debug builds by default, which can cause issues with embedding necessary files into the binary. Therefore, make sure to use the `--release` flag when building your Rust application for use inside Docker.
+
```

Comment:
Actually it should not be completely necesary to build it with `--release` flag, but alternatively you can add a cargo `debug-embed` feature flag to the `utoipa-swagger-ui` dependency seen here: https://github.com/juhaku/utoipa/tree/master/utoipa-swagger-ui#crate-features. That will enable the RustEmbed's `debug-embed` feature which allows debug builds to embed files as well.

Reply:
I think there could also be a link to the `utoipa-swagger-ui`'s REAMDE.md as well

Reply:
Hmm ... maybe you are right. ^^"

I would at least describe it more, especially with the wording Docker. Because I think some other people will fall into the same kind of problem. ^^"